diff --git a/node_modules/react-native/ReactAndroid/cmake-utils/ReactNative-application.cmake b/node_modules/react-native/ReactAndroid/cmake-utils/ReactNative-application.cmake
index 805ef11..0da2cb1 100644
--- a/node_modules/react-native/ReactAndroid/cmake-utils/ReactNative-application.cmake
+++ b/node_modules/react-native/ReactAndroid/cmake-utils/ReactNative-application.cmake
@@ -33,12 +33,18 @@ endif(CCACHE_FOUND)
 set(BUILD_DIR ${PROJECT_BUILD_DIR})
 if(CMAKE_HOST_WIN32)
         string(REPLACE "\\" "/" BUILD_DIR ${BUILD_DIR})
+        string(REPLACE "\\" "/" REACT_ANDROID_DIR ${REACT_ANDROID_DIR})
 endif()
 
 file(GLOB input_SRC CONFIGURE_DEPENDS
         ${REACT_ANDROID_DIR}/cmake-utils/default-app-setup/*.cpp
         ${BUILD_DIR}/generated/autolinking/src/main/jni/*.cpp)
 
+# Ensure that input_SRC paths use forward slashes
+foreach(path IN LISTS input_SRC)
+    string(REPLACE "\\" "/" path "${path}")
+endforeach()
+
 add_library(${CMAKE_PROJECT_NAME} SHARED ${input_SRC})
 
 target_include_directories(${CMAKE_PROJECT_NAME}
